### DISCUSSION TITLE
Using Sqlite browser to access - "Notes" on Mac

### DISCUSSION THREAD

**Discussion by Darkersound** [sqlitebrowser/sqlitebrowser#2967] [Q&A]
I've been trying to work out how to open my Mac "notes" (NoteStore.sqlite) database to see when I originally "created" a note.

I mistakenly since accessed this note and I have altered the original note meaning the file creation shown date is changed when I view this within Notes.

I am hoping that if I access the above database file, I will be able to view the original file creation date.

Can anyone assist or direct?

Many thanks mate in advance (apologies I am not a programmer).

Russ

**Comment by chrisjlocke** [comment]
I'm not a Mac owner so haven't tried opening the noteStore database.  There's no guarantee it'll contain date information on when records were added or edited.
If you want to email me your .sqlite file I can have a look, but I assume it'll contain personal information, so may not be a good idea.
At worst, if you open DB4S, then open the noteStore.sqlite database, you'll see a selection of tables on the left.  You just need to go to the second tab (Browse Data) and by selecting various tables, you should see the note you're after.
If you get that far and still struggle, you can take screenshots and paste them back here and we can see if we can assist further.
